Source/WebKit/UIProcess/WebPageProxy.messages.in

```diff
@@ -647,4 +647,6 @@ CreateAppHighlightInSelectedRange)
     HasActiveNowPlayingSessionChanged(bool hasActiveNowPlayingSessionChanged)
 
     SetAllowsLayoutViewportHeightExpansion(bool newValue)
+
+    SetPageScaleFactor(float scaleFactor, int x, int y) -> ()
```

Comment:
We should consider passing a `WebCore::IntPoint origin` instead of x / y

Reply:
Seems unfortunate to be added new WebPageProxy IPC just for testing. I think the IPC should be in the TestRunner instead and the API should be in TestRunner.idl instead.